hummingbot/core/time_iterator.pxd

```diff
@@ -8,6 +8,7 @@ cdef class TimeIterator(PubSub):
     cdef:
         double _current_timestamp
         Clock _clock
+        double _overriding_tick
```

Comment:
Would it not be possible to override the TimeIterator _clock tick_size instead?

Reply:
you meant **_clock.tick_size = _overriding_tick**?
**time_iterator**'s **_clock** is a clock that has the time_iterator as a **_child_iterators**. The _clock is not exclusive for the time_iterator.
So, **_clock.tick_size = _overriding_tick** will affect all other iterators's tick that are in clock's iterators.

Reply:
Right, the whole point of it!, lol, silly me